templates/frontend/objects/announcement_full.tpl

```diff
@@ -13,23 +13,23 @@
 
 <article class="obj_announcement_full">
 	<h1>
-		{$announcement->getLocalizedTitle()|escape}
+		{$announcement->getLocalizedData('title')|escape}
 	</h1>
 	<div class="date">
-		{$announcement->getDatePosted()|date_format:$dateFormatShort}
+		{$announcement->getAttribute('datePosted')|date_format:$dateFormatShort}
 	</div>
-	{if $announcement->getImage()}
+	{if $announcement->getData('image')}
 		<img
 			class="obj_announcement_full_image"
-			src="{$announcement->getImageUrl()}"
-			alt="{$announcement->getImageAltText()}"
+			src="{$announcement->getAttribute('imageUrl')}"
```

Comment:
Is it possible to just access `$announcment->imageUrl` rather than using `getAttribute`? That seems to be the more common Eloquent formulation, and what we've used elsewhere. (Is it necessary for the composed attributes or something?)

Reply:
It's possible. I simple don't like getting those magically. But that's what Laravel uses in their examples and don't mind adopting it.